### PR TITLE
Reformat + simplify java code from bronze modules

### PR DESCRIPTION
**Pull request by nathangong**
I wanted to make code from some bronze modules easier to read by adding Kattio instead of buffered reader and formatting the code to follow Java conventions. The reformatted code has all been tested.

### CODE REVIEW

File: content/2_Bronze/Complete_Rec.mdx [hunk 1]
```diff
@@ -599,34 +437,38 @@ public class Main {
 		}
 		for (int i = 0; i < 26; ++i) {
 			if (cnt[i] > 0) {
-				--cnt[i];
-				perm += (char)(i + 'a');
+				cnt[i]--;
```
Comment (bqi343) [inline content/2_Bronze/Complete_Rec.mdx:440]: why?

**Review by bqi343 — APPROVED** [review]
looks good

File: content/2_Bronze/Complete_Rec.mdx [hunk 2]
```diff
@@ -1057,15 +900,15 @@ public class Main {
 	}
 
 	public static void main(String[] args) throws Exception {
-		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		Kattio io = new Kattio();
```
Comment (maggie-j-liu) [inline content/2_Bronze/Complete_Rec.mdx:903]: add Kattio Codesnip?

File: content/2_Bronze/Complete_Rec.mdx [hunk 3]
```diff
@@ -930,11 +772,11 @@ public class Chessboard {
 	public static ArrayList < Integer > perm = new ArrayList < Integer > ();
 	public static boolean[] chosen = new boolean[8];
 	public static int ans = 0;
+
 	public static void main(String[] args) throws IOException {
-		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
-		PrintWriter out = new PrintWriter(System.out);
+		Kattio io = new Kattio();
```
Comment (maggie-j-liu) [inline content/2_Bronze/Complete_Rec.mdx:777]: add kattio codesnip?

File: content/2_Bronze/Complete_Rec.mdx [hunk 4]
```diff
@@ -599,34 +437,38 @@ public class Main {
...
+		io.println(perms.size());
 		for (String i : perms) {
-			System.out.println(i);
+			io.println(i);
 		}
-		br.close();
+		io.close();
 	}
+
+    CodeSnip{Kattio}
```
Comment (maggie-j-liu) [inline content/2_Bronze/Complete_Rec.mdx:471]:
```suggestion
	CodeSnip{Kattio}
```

File: content/2_Bronze/Complete_Rec.mdx [hunk 5]
```diff
@@ -77,7 +77,7 @@ ll solve(int i, ll s1, ll s2){
 		return abs(s1 - s2);
```
Comment (maggie-j-liu) [inline content/2_Bronze/Complete_Rec.mdx:77]: i also modified this file in pr 1652, hopefully the conflicts won't be too bad...

File: content/2_Bronze/Rect_Geo.mdx [hunk 6]
```diff
@@ -233,10 +233,12 @@ public class Billboard {
...
 		}
-		PrintWriter out = new PrintWriter(new FileWriter("billboard.out"));
-		out.println(ans);
-		out.close();
+
+		io.println(ans);
+		io.close();
 	}
+
+    CodeSnip{Kattio}
```
Comment (maggie-j-liu) [inline content/2_Bronze/Rect_Geo.mdx:241]:
```suggestion
	CodeSnip{Kattio}
```

File: content/2_Bronze/Simulation.mdx [hunk 7]
```diff
@@ -82,41 +82,45 @@ int main()
...
-	public static void main(String['Simulation'] args) throws IOException
-	{
-		io=new Kattio("shell");
-		for(int i=0;i<3;++i)
-			shell_at_position[i]=i; // We arbitrarily place the shells down (In this case, I happen to be putting the shell with label i at position i)
-		N=io.nextInt();
-		for(int i=0;i<N;++i)
-		{
+public class Shell throws IOException {
+	public static void main(String[] args) throws {
```
Comment (maggie-j-liu) [inline content/2_Bronze/Simulation.mdx:87]: wait idrk java but shouldn't it be
```suggestion
public class Shell {
	public static void main(String[] args) throws IOException {
```

File: content/2_Bronze/Simulation.mdx [hunk 8]
```diff
@@ -82,41 +82,45 @@ int main()
...
+		}
+
+		for (int i = 0; i < N; i++) {
 			// Zero indexing: offset all positions by 1
-			int a=io.nextInt()-1;
-			int b=io.nextInt()-1;
-			int g=io.nextInt()-1;
+			int a = io.nextInt()-1;
+			int b = io.nextInt()-1;
+			int g = io.nextInt()-1;
```
Comment (maggie-j-liu) [inline content/2_Bronze/Simulation.mdx:109]:
```suggestion
			int a = io.nextInt() - 1;
			int b = io.nextInt() - 1;
			int g = io.nextInt() - 1;
```

File: content/2_Bronze/Simulation.mdx [hunk 9]
```diff
@@ -214,39 +219,46 @@ int main()
...
+	static int N = 3;
+	static int[] capacity = new int[N];
+	static int[] milk = new int[N];
+
+	public static void pour(int i, int j) {
+		// Amount of milk to pour is the minimum of remaining milk in bucket i
+		// and the available capacity in bucket j
+		int amount = Math.min(milk[i], capacity[j] - milk[j]);
+		milk[i] -= amt;
+		milk[j] += amt;
```
Comment (maggie-j-liu) [inline content/2_Bronze/Simulation.mdx:238]:
```suggestion
		int amount = Math.min(milk[i], capacity[j] - milk[j]);
		milk[i] -= amount;
		milk[j] += amount;
```
did you test this code??

**Review by maggie-j-liu — CHANGES_REQUESTED** [review]

File: content/2_Bronze/Simulation.mdx (hunk 9 above)
Comment (nathangong) [reply content/2_Bronze/Simulation.mdx:238]: Oh shoot yeah I decided to change the variable right after I tested it and didn't catch that thanks

**Review by maggie-j-liu — APPROVED** [review]

**Pull request merged**